csv2Urls.py

```diff
@@ -1,13 +1,16 @@
 import csv
 
+#path to csv
 csv_Path = 'eis-listing.csv'
```

Comment:
Can we move this to a variable or something? again just thinking about abstracting out the EIS/EPA stuff